xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-plugins/src/main/webjar/xwiki-resource/resourcePicker.css

```diff
@@ -19,6 +19,19 @@
  */
 #template('colorThemeInit.vm')
 
+/* We need to redefine this class to have better priority than the CKEditor CSS reset. 
+Without this higher priority redefinition, the elements with this class are still shown. */
```

Comment:
It is not exactly clear to what is redefined, and why exactly this redefinition is needed.